src/options.ts

```diff
@@ -184,7 +184,7 @@ export interface Options {
    *
    * @example
    * // With removeNodeProtocol enabled:
-   * import('node:fs'); // becomes import('fs')
+   * import('node:fs'); // becomes import('node')
```

Comment:
That's wrong, removing the `node:` prefix from `node:fs` should give `fs`. If it doesn't work that way, it's a bug.